tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -183,10 +183,6 @@ internal TracerSettings(IConfigurationSource? source, IConfigurationTelemetry te
                                   .WithKeys(ConfigurationKeys.TracerMetricsEnabled)
                                   .AsBool(defaultValue: false);
 
-            StatsComputationEnabledInternal = config
-                                     .WithKeys(ConfigurationKeys.StatsComputationEnabled)
-                                     .AsBool(defaultValue: false);
-
```

Comment:
I don't think you meant to delete this? 🤔

Reply:
moved below, it's dependent on `IsRunningInAzureFunctionsConsumptionPlan` && `IsRunningInGCPFunctions`